modules/twinklexfd.js

```diff
@@ -474,7 +474,7 @@ Twinkle.xfd.callbacks = {
 			// Notification to first contributor
 			var wikipedia_page = new Morebits.wiki.page(mw.config.get('wgPageName'));
 			wikipedia_page.setCallbackParameters(pageobj.getCallbackParameters());
-			if (mw.config.get('wgPageContentModel') !== 'Scribunto') {
+			if (['Scribunto', 'javascript', 'sanitized-css', 'css'].indexOf(mw.config.get('wgPageContentModel')) === -1) {
```

Comment:
根據 https://github.com/Xi-Plus/twinkle/issues/226#issuecomment-854267121
```suggestion
			if (mw.config.get('wgPageContentModel') === 'wikitext') {
```
不就好了？